nesta/core/schemas/tier_1/mappings/defaults/defaults.json

```diff
@@ -1,24 +1,24 @@
 {
-    "mappings":{
-	"_doc":{
```

Comment:
(just a comment, in Svizzle I'm actually going back to tabs as it's editor configurable so it makes everyone happy :)

Reply:
oops, actually I have a new unit test that this would have failed (all json in the repo must be clean to pass) - so this will fail anyway :)